rest-api/src/main/java/gov/cms/qpp/conversion/api/controllers/v1/CpcFileControllerV1.java

```diff
@@ -24,12 +15,21 @@
 import org.springframework.web.bind.annotation.RequestMapping;
 import org.springframework.web.bind.annotation.RestController;
 
+import gov.cms.qpp.conversion.api.model.Constants;
+import gov.cms.qpp.conversion.api.model.CpcFileStatusUpdateRequest;
+import gov.cms.qpp.conversion.api.model.UnprocessedCpcFileData;
+import gov.cms.qpp.conversion.api.services.CpcFileService;
+import gov.cms.qpp.conversion.util.EnvironmentHelper;
+
+import java.io.IOException;
+import java.util.List;
+
 /**
  * Controller to handle cpc file data
  */
 @RestController
 @RequestMapping("/cpc")
-@CrossOrigin
+@CrossOrigin(allowCredentials = "true")
```

Comment:
What does this change?

Reply:
That allows cookies to be passed cross origin.  In Spring 4.x, if this was unspecified, it defaulted to true.  In Spring 5.x it defaults to false and broke our SimpleUI.  So, we now explicitly say we allow cookies.